scripts/test/runtests.ps1

```diff
@@ -44,7 +49,7 @@ pushd "$TestBinRoot"
 
 # Run each test project
 $TestProjects | ForEach-Object {
-    & "corerun.exe"  "xunit.console.netcore.exe" "$_.dll" -xml "$_.xml" -notrait category=failing
+    & "corerun.exe"  "xunit.console.netcore.exe" "$_.dll" -xml "$_.xml" -notrait category=failing -parallel none
```

Comment:
I was having some issues with tests so I preventively turned it off. However it turned out my tests were failing because the stage 0 was used and not stage 2 :)

I left them turned off as a safety measure in case tests from different test classes might read / write on the same paths which would lead to races and weird test failures.
Until CLI tests use a shared mechanism that guarantees the absence file IO races, I think it's safer to not have parallel tests.